src/styles/button.scss

```
@@ -0,0 +1,49 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  box-sizing: border-box;
+  padding: 0;
+  border: 0;
+
+  cursor: pointer;
+
+  &--primary-buy {
+    width: 102px;
+    height: 48px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 15px;
+    color: #fff;
```

don't forget to create variables for all colors
```suggestion
    color: $c-white;
```